survey_sale_generation/models/sale_order.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    survey_user_input_id = fields.Many2one(comodel_name="survey.user_input", copy=False)
```

Comment:
I think the user should not be able to change this field, to maintain data integrity. What do you think?

```suggestion
    survey_user_input_id = fields.Many2one(comodel_name="survey.user_input", copy=False, readonly=True)
```

Reply:
Now in view